src/main/scala/viper/silver/ast/utility/Consistency.scala

```diff
@@ -29,7 +29,7 @@ object Consistency {
     recordIfNot(suspect, !property, message)
 
   /** Names that are not allowed for use in programs. */
-  def reservedNames: Seq[String] = FastParser.keywords.toSeq
+  def reservedNames: Seq[String] = FastParserCompanion.basicKeywords.toSeq
```

Comment:
Sequences guarantee a specific traversal order, sets don't, which may result in user-observable nondeterminism. Most of Viper's codebase tries to avoid collections with nondeterministic traversal orders.

Reply:
Ah damn, I just changed it to a Set, I'll change it back to a sequence then ^^

Reply:
Otherwise, we could use a `SortedSet`

Reply:
Ah yes, that would be a good compromise. I'll do that later, this PR actually just modified this line and didn't actually introduce it, so this is kind of orthogonal to the global state thing.